packages/truffle-core/lib/commands/config.js

```diff
@@ -0,0 +1,41 @@
+const command = {
+  command: "config",
+  description: "set user-level configuration options",
+  config: {
+    usage: "truffle config --enable-analytics",
```

Comment:
The value for usage should probably be something like `truffle config <analytics_option>` if we are following the conventions in the command files, right?  Are we going to eventually add other options in here for this command?  We might want to consider having a format like `truffle config --analytics off|on`...what do you think?